yw_website/yw_website/apps/website/models.py

```diff
@@ -108,6 +108,11 @@ class Meta:
     in_program_block = models.ForeignKey("self", on_delete=models.DO_NOTHING, null=True)
     run = models.ForeignKey(Run, on_delete=models.CASCADE)
 
+    def __str__(self):
+        s = "programblock_id: " + str(self.programblock_id)
```

Comment:
I would prefer using .format to add variables to the string as it's a bit easier to read and can be a one-liner, this suggestion applies to all of the __str__() methods

Reply:
I'll take a look. The str() method was mostly for me to be able to view some relevant info but it can also be pulled from the database using a viewer like Datum